src/ProjectManagement/Utility/PathUtility.cs

```diff
@@ -91,7 +91,8 @@ public static string GetPath(Uri uri)
         public static string GetPackagePath(string path)
```

Comment:
@zhili1208, Can you please git blame this file to understand the history of changes on this method? As I remember it, this method was changed more than once for fixing 2 or more different bugs. Please make sure that your fix does not cause any regressions